acceptance/api/v1/application_portforward_test.go

```diff
@@ -92,16 +92,19 @@ var _ = Describe("AppPortForward Endpoint", func() {
 			})
 		})
 
-		When("you you specify a specific instance", func() {
+		When("you specify a specific instance", func() {
 			var conn httpstream.Connection
 			var connErr error
 
+			// Use separate application instead of the main of the suite
+			appName := "portforward"
```

Comment:
```
This detail is really important. Ginkgo requires, by default, that specs be fully independent. This allows Ginkgo to shuffle the order of specs and run specs in parallel. We'll cover this in more detail later on but for now embrace this takeaway: "Declare in container nodes, initialize in setup nodes".
```
from here: https://onsi.github.io/ginkgo/#extracting-common-setup-beforeeach

Reply:
Extended comment with `Bug fix:` prefix, and also added the link.

Reply:
Done, commit e713fc21cfce6b2f7905a1577a64640418d41899

Reply:
sorry, what I meant was that it's bad practice to initialize variables in "container" blocks (when, describe, etc). This is explained in the link I sent. This initialization should be moved inside the before block.

Reply:
Ok.

Reply:
Done, 0d2335dad901577d605fdb1aa441c88a93533a02